bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/api/SamsungTvService.java

```diff
@@ -42,15 +44,15 @@
      * Procedure for register event listener.
      *
      * @param listener
-     *            Event listener instance to handle events.
+     *                     Event listener instance to handle events.
      */
```

Comment:
this looks wrong. can you use the formatter? In a newly installed IDE this is not properly initialized, go to Windows/Preferences/Java/Codestyle/Formatter, Select something different than ESH, apply, re-select ESH, apply. Then open and save this file. It should get the correct formatting.

Reply:
Sure, I reformatted all files with the ESH codestyle formatter properly enabled.